Dockerfile

```diff
@@ -9,5 +9,4 @@ RUN gradle --no-daemon clean build bintrayUpload \
     -Prelease_version=${release_version} \
     -Pbintray_user=${bintray_user} \
     -Pbintray_key=${bintray_key} \
-    -Pvcs_url=${vcs_url}
-
+    -Pvcs_url=${vcs_url}
```

Comment:
note: it looks like most of the parameters can be removed from the Dockerfile

Reply:
Current .github workflow doesn't use Dockerfile for publishing maven artifact. Maybe remove Dockerfile and .gitlab-ci files?

Reply:
Let's continue discussion in https://github.com/th2-net/th2-codec/pull/68 PR